packages/engine/src/lib/services/storage.service.ts

```diff
@@ -53,8 +56,12 @@ export const createStorageService = ({
 
       try {
         const sizeOfValue = sizeof(request.value);
-        if (sizeOfValue > STORE_VALUE_MAX_SIZE) {
-          throw new StorageLimitError(request.key, STORE_VALUE_MAX_SIZE);
+        const maxStorageSizeMB =
```

Comment:
minor: we can extract it to a var like it was before (here and not in the package), just to avoid the calculation each time and shorten the code here